certificate-policy.md

```diff
@@ -1704,21 +1759,51 @@ The CA's audit SHALL be performed by a Qualified Auditor. A Qualified Auditor me
 1. Independence from the subject of the audit;
 2. The ability to conduct an audit that addresses the criteria specified in an Eligible Audit Scheme (see Section 8.1);
 3. Employs individuals who have proficiency in examining Public Key Infrastructure technology, information security tools and techniques, information technology and security auditing, and the third-party attestation function;
-4. (For audits conducted in accordance with any one of the ETSI standards) accredited in accordance with ISO 17065 applying the requirements specified in ETSI EN 319 403;
-5. (For audits conducted in accordance with the WebTrust standard) licensed by WebTrust;
-6. Bound by law, government regulation, or professional code of ethics; and
-7. Except in the case of an Internal Government Auditing Agency, maintains Professional Liability/Errors & Omissions insurance with policy limits of at least one million US dollars in coverage
+4. (For audits conducted in accordance with the WebTrust standard) licensed by WebTrust;
+5. Bound by law, government regulation, or professional code of ethics;
+6. Except in the case of an Internal Government Auditing Agency, maintains Professional Liability/Errors & Omissions insurance with policy limits of at least one million US dollars in coverage; and
+7. If the CA chooses to obtain a WebTrust audit, the CA SHALL use a WebTrust licensed Auditor to perform the audit. 
 
 ## 8.3 Assessor's relationship to assessed entity
 
 ## 8.4 Topics covered by assessment
-The CA SHALL undergo an audit in accordance with one of the following schemes:
-
-1. WebTrust for Certification Authorities v2.0;
-2. A national scheme that audits conformance to ETSI TS 102 042 / ETSI EN 319 411-1; or
-4. If a Government CA is required by its Certificate Policy to use a different internal audit scheme, it MAY use such scheme provided that the audit either (a) encompasses all requirements of one of the above schemes or (b) consists of comparable criteria that are available for public review.
-
-Whichever scheme is chosen, it MUST incorporate periodic monitoring and/or accountability procedures to ensure that its audits continue to be conducted in accordance with the requirements of the scheme.
+All CAs SHALL undergo an audit in accordance with one of the following schemes:
+
+WebTrust Option
+Required for all WebTrust and Time Stamping
+1. WebTrust for Certification Authorities v2.0 w/ Federal PKI Addendum Letter
+
+Additional
+2. WebTrust for Certification Authorities - SSL Baseline with Network Security V2.2 (Required for Server Authentication)
+3. WebTrust for Certification Authorities - Code Signing Certificates v1.0 (Required for Code Signing)
+
+Federal PKI Option
+1. Federal PKI Annual Audit Requirements
+
+**Note**: If the CA chooses the Federal PKI Audit option the issuing CAs SHALL be constrained to issue certificates only to U.S. Government controlled domains.
```

Comment:
This seems like not what we've discussed. All issuing CAs on this root, and potentially the root itself, will be constrained to issue certificates only to USG controlled domains -- regardless of the audit in use.

In addition, I don't recall discussing (and it's possible I've missed this) allowing USG-internal audits. Third party audits are standard fare and should be part of how we're building trust with the non-USG community.